index.html

```diff
@@ -35,7 +35,13 @@ <h1 class="logo__text">Renome</h1>
                             <span></span>
                         </div>
                         <nav class="header__menu" id="header_menu">
-                            <ul class="header__list">
+                            <ul class="header__list" id="header_list">
+                                <li>
+                                    <form action="/search/" target="_blank" class="form-search">
```

Comment:
Where did you get an action?
Why target is blank?

Reply:
It is just a dummy - redirecting to a blank page (this can be replaced to any other action)

Reply:
You project not suppose to have any backend and no form will be sent to anywhere so remove action
having target as a _blank with no purpose is bad practice https://css-tricks.com/use-target_blank/

Reply:
Ok, deleted it